roles/nginxplus/files/fail2ban/nginx-f_inclusive.conf

```
@@ -0,0 +1,4 @@
+nginx-f_inclusive.conf
+[Definition]
+failregex = ^<HOST> .+?([&?]f(_inclusive)?%%5B.*?){8,}
```

```suggestion
failregex = ^<HOST> .+?([&?]f(_inclusive)?%5B.*?){8,}
```

I think we may only need one `%`

---

Scratch that, I think we actually need something like this since access.log is a json log:

```suggestion
failregex = ^\{\"remote_ip\"\: \"<HOST>\".*?\"uri\"\:.+?([\&]?f(_inclusive)?%%5B.*?){8,}
```